terms.html

```diff
@@ -71,7 +72,7 @@
 </ul>
 <p>in any applicable law (which may depend on where you are located and any jurisdiction to which your activities are directed).</p>
 <h2>Use of cookies</h2>
```

Comment:
We also use third party services, which set cookies.

Mixpanel and Stripe both set cookies on PDFTables.com. I'm a bit surprised that Intercom hasn't for me - I assumed it would as well.

We also may use other such services in the future.

Does this clause cover that, and also do we need anything else to make clear we use such third party services? (And indeed AWS!)